website/docs/guides/ha_controllers.markdown

```diff
@@ -0,0 +1,126 @@
+---
+layout: "appgatesdp"
+page_title: "high availability controllers"
+sidebar_current: "docs-appgatesdp-guide-high_availability_controllers"
+description: |-
+  Provisioning high availability controllers
+---
+
+## Provisioning high availability controllers
+
+
+In appliance >= 5.4.0 we cant create more then 1 controller at the time,
+it means that we must first create an appliance, seed it, then activate the controller function.
```

Comment:
Just to clear things up, you never could do that as joining/leaving the cluster can only be done 1 node at a time.
It was allowed by the API but would result in errors.